mmcls/datasets/pipelines/transforms.py

```diff
@@ -1122,10 +1122,9 @@ def __call__(self, results):
 
         results = self.aug(**results)
 
-        if 'gt_labels' in results:
```

Comment:
In my case, this can't solved the problem, the shape is not equal after `results['gt_label'].astype(np.int64)`

In my lastest commit, I using `copy.deepcopy` to backup the original `gt_label` and update it to dict `results` after aug. I think this way we can make sure it is the same.